### PR TITLE
Add comment

### PR DESCRIPTION
**Pull request by CodelingoBotTester**

### CODE REVIEW

File: example.php [hunk 1]
```diff
@@ -4,4 +4,5 @@
 // is_null is a function that does stuff 
 is_null($foo);
 
-?>
\ No newline at end of file
+// Some comment
```
Comment (CodelingoBotTester) [inline example.php:7]:
```diff
- // is_null is a function that does stuff 
+ // is_null {{ .funcComment }}
```

Change `{{ .funcComment }}` by commenting: `@CodeLingoBot rewrite set funcComment "your comment"`

Comment (Daanikus) [reply example.php]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (Daanikus) [reply example.php]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (Daanikus) [reply example.php]: @CodelingoBotTester yo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester yo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester yo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester yo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester yo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester yo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester yo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester yo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester yo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester yo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester ypo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester ypo

Comment (Daanikus) [reply example.php]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (Daanikus) [reply example.php]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (Daanikus) [reply example.php]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (Daanikus) [reply example.php]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (Daanikus) [reply example.php:7]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (Daanikus) [reply example.php:7]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply example.php:7]: value of variable rewrite must be quoted

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `@CodelingoBotTesterTester rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Comment (Daanikus) [reply example.php:7]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `@CodelingoBotTesterTester rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Comment (Daanikus) [reply example.php:7]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply example.php:7]: value of variable rewrite must be quoted

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `@CodelingoBotTesterTester rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Comment (Daanikus) [reply example.php:7]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply example.php:7]: value of variable rewrite must be quoted

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `@CodelingoBotTesterTester rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Comment (Daanikus) [reply example.php:7]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply example.php:7]: GET https://api.github.com/repos/Daanikus/interactive-rewrite-test/git/refs/heads/rewrite: 404 Not Found []

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `CodelingoBotTester rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester to reach us.

</details>

Comment (Daanikus) [reply example.php:7]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (Daanikus) [reply example.php:7]: hi @CodelingoBotTester

Comment (Daanikus) [reply example.php:7]: hi @CodelingoBotTester

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodelingoBotTester commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester review`</summary>

    + `@CodelingoBotTester review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester rewrite`</summary>

    + `CodelingoBotTester rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester to reach us.

</details>

Comment (Daanikus) [reply example.php:7]: @CodelingoBotTester rewrite set funcName "Some comment"

Comment (Daanikus) [reply example.php:7]: @CodeLingoBot rewrite set funcName "Some comment"

Comment (Daanikus) [reply example.php:7]: @CodeLingoBot rewrite set funcComment "Some comment"

Comment (Daanikus) [reply example.php:7]: @CodelingoBotTester rewrite set funcComment "Some comment"

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodeLingoBot to reach us.

</details>

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodeLingoBot to reach us.

</details>

Comment (Daanikus) [reply example.php:7]: @CodelingoBotTester rewrite set funcComment "Some comment"

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

Comment (CodelingoBotTester) [reply example.php:7]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>

    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>

    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.



</details>

**Pull request closed without merging**